### ISSUE TITLE
Problem when submit a PUT

### ISSUE THREAD

**Issue by rafamarts** [flaviostutz/schellar#9]
When we submit a PUT we could identify the schedule name in URL (required) and in the body JSON. But if accidentally send different names in those both fields, the mongo will make the change in the body one. Seems that the code is ignoring the URL param.

Could I take it?

**Comment by flaviostutz** [comment]
Sure!

I think when those two parameters are different you should return an error 4xx. What do you think?

**Comment by rafamarts** [comment]
I agree with you. We shouldn't update anything on db, because it is a malformed request.

I suggest to return 422, according [RFC4918](https://tools.ietf.org/html/rfc4918):

"11.2.  422 Unprocessable Entity

   The 422 (Unprocessable Entity) status code means the server
   understands the content type of the request entity (hence a
   415(Unsupported Media Type) status code is inappropriate), and the syntax of the request entity is correct (thus a 400 (Bad Request)
   status code is inappropriate) but was unable to process the contained
   instructions.  For example, this error condition may occur if an XML
   request body contains well-formed (i.e., syntactically correct), but
   semantically erroneous, XML instructions"

What do you think?
